Makefile

```diff
@@ -122,7 +122,14 @@ unit: $(CARGO_TARGET_DIR)
 	$(CARGO) test
 
 .PHONY: integration
+# The TEST_PLUGINS envvar is used by bats to specify path to test-plugins
+# directory
+ifdef TEST_PLUGINS
+$(info Skipping examples build as TEST_PLUGINS is set)
+integration: $(CARGO_TARGET_DIR)
+else
 integration: $(CARGO_TARGET_DIR) examples
+endif
```

Comment:
Maybe we should just drop the `examples` dependency instead of making it more complicated. We do not depend on `build` either so it is not much different.